taskvine/src/examples/vine_example_peer_transfer.c

```diff
@@ -24,58 +12,51 @@ with all the same tasks on the worker.
 #include <errno.h>
 #include <unistd.h>
 
-const char *query_string = ">P01013 GENE X PROTEIN (OVALBUMIN-RELATED)\n\
-		QIKDLLVSSSTDLDTTLVLVNAIYFKGMWKTAFNAEDTREMPFHVTKQESKPVQMMCMNNSFNVATLPAE\n\
-		KMKILELPFASGDLSMLVLLPDEVSDLERIEKTINFEKLTEWTNPNTMEKRRVKVYLPQMKIEEKYNLTS\n\
-		VLMALGMTDLFIPSANLTGISSAESLKISQAVHGAFMELSEDGIEMAGSTGVIEDIKHSPESEQFRADHP\n\
-		FLFLIKHNPTNTIVYFGRYWSP\n\
-";
-
-#define BLAST_URL "https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-2.13.0+-x64-linux.tar.gz"
+#define BLAST_URL "https://mirrors.slackware.com/slackware/slackware-iso/slackware-12.0-iso/slackware-12.0-install-d1.iso"
```

Comment:
I suggest creating a local large file with `dd` and make that the input to the tasks.  The first workers will get the file from the manager, but others would get it from the peers.  (We do not want to mess up with the download stats of slackware, or cause unnecessary traffic of large files.)